static/src/javascripts/boot.js

```diff
@@ -33,6 +35,11 @@ define([
     var guardian = window.guardian;
     var config = guardian.config;
 
+    // global variable for sharing modules across bundles
+    guardian.app = {
```

Comment:
Just in case, you might want to make sure there is not an existing `app` key already.

```
guardian.app = guardian.app || {};
```